.gitignore

```diff
@@ -0,0 +1,5 @@
+print/attendees.txt
+print/cookies.txt
+print/valid-attendees.txt
+.vagrant
```

Comment:
.vagrant is temp files created by [Vagrant](http://www.vagrantup.com.com).  Thought I had another PR that added a Vagrantfile in scalereg/ and scalereg/print/ to allow for easy local testing on the various distros we support.  We're using that for all the chef bits these days as well.